install.sh

```
@@ -0,0 +1,247 @@
+main ()
+{
+    lint_dir=$( read_link_f "$0" )
+    target_dir="$1"
+
+    cd "$target_dir"
+
+    uses_typescript=$(
+        npm ls typescript > /dev/null 2>&1 && echo "1" || echo ""
+    )
+
+    printf "${ansi_bold}INSTALL NPM PACKAGES${ansi_reset}\n"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)